notifications_utils/logging.py

```diff
@@ -134,31 +142,37 @@ def get_handler(app):
     if app.debug:
         # Human readable stdout logs that omit static route 200 responses
         logging.getLogger('werkzeug').addFilter(is_200_static_log)
-        stream_handler.setFormatter(logging.Formatter(LOG_FORMAT, TIME_FORMAT))
-    else:
-        # Machine readable json to stdout
-        stream_handler.setFormatter(JsonFormatter(LOG_FORMAT, TIME_FORMAT))
+
+    stream_handler.setFormatter(
+        JsonFormatter(
+            CELERY_LOG_FORMAT if _service_map.get(app.name, API_LOG_FORMAT) == 'celery' else API_LOG_FORMAT,
+            TIME_FORMAT,
+        )
+    )
 
     return stream_handler
 
 
 class AppNameFilter(logging.Filter):
     def __init__(self, app_name):
-        self.app_name = app_name
+        self.service = _service_map.get(app_name, 'test')
 
     def filter(self, record):
-        record.application = self.app_name
+        record.application = self.service
         return record
 
 
 class RequestIdFilter(logging.Filter):
-    @property
-    def request_id(self):
-        if has_request_context() and hasattr(request, 'request_id'):
-            return request.request_id
-
-        return 'no-request-id'
-
     def filter(self, record):
-        record.requestId = self.request_id
+        # The else is for celery
+        if not getattr(record, 'requestId', ''):
+            record.requestId = RequestIdFilter._get_api_id() if has_request_context() else 'no-request-id'
         return record
+
+    @classmethod
+    def _get_api_id(self) -> str:
```

Comment:
nit: usually the first arg of a classmethod is cls, not self. just something to consider.